### PR TITLE
Add the possibility to add comments to the YouTube channels

### PR DESCRIPTION
**Pull request by hruntov**

### CODE REVIEW

File: youtube_base/youtubers/admin.py [hunk 1]
```diff
@@ -33,6 +33,19 @@ class CategoryAdmin(admin.ModelAdmin):
                     'description')
 
 
+class CommentAdmin(admin.ModelAdmin):
+    """The CommentAdmin class defines the admin interface for the Comment model."""
+    list_display = ('name',
```
Comment (Yevhenii-Yatchenko) [inline youtube_base/youtubers/admin.py:38]: Why not to inline several tuple items as a single line?

File: youtube_base/users/fixtures/users.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
+{
+    "model": "auth.user",
+    "pk": 1,
+    "fields": {
+        "password": "pbkdf2_sha256$720000$7hSbPSNNxspzVTSszBFdi7$H6OCTph+clt7d8pFTbALAsLj/xVvo20Jndgsz3NQiZ4=",
```
Comment (Yevhenii-Yatchenko) [inline youtube_base/users/fixtures/users.json:6]: this is dummy data for tests, right?

Comment (hruntov) [reply youtube_base/users/fixtures/users.json:6]: Yes. It's hashed 'test_password'.

File: youtube_base/youtubers/admin.py (hunk 1 above)
Comment (hruntov) [reply youtube_base/youtubers/admin.py:38]: I just want to that the file has the same style.

**Pull request merged**